doc/specs/stdlib_logger.md

```diff
@@ -34,6 +34,10 @@ The logger variables have the option to:
 * indent subsequent lines of the messages; and
 * format the text to fit within a maximum column width.
 
+While every effort has been made to make the code thread and
+asynchronous I/O safe, it is always best to have each process write to
+its own dedicated logger file.
```

Comment:
```suggestion
While every effort has been made to make the code process and
asynchronous I/O safe, it is always best to have each process write to
its own dedicated logger file.
For thread parallelism (e.g., with OpenMP), it is advised to put the logger call in a guarding region (e.g., in an OpenMP critical region.
```

Related to [this](https://github.com/fortran-lang/stdlib/pull/245#issuecomment-732691050) and [this](https://github.com/fortran-lang/stdlib/pull/245#issuecomment-732718840) comment.

@wclodius2 @14NGiestas @milancurcic We usually speek about "process" when related to MPI process. Thread parallelism is related to OpenMP and Pthreads.

Reply:
I have revised the discussion as suggested, adding a closing parenthesis at the end.